components/Footer.tsx

```diff
@@ -7,13 +7,23 @@ const Footer = () => (
     <div className="container">
       <div className="footer-content">
         <div className="footer-logo">
-          <img src="/images/footer-logo.svg" alt="footer-logo" />
+          <img
+            src="/images/footer-logo.svg"
+            alt="footer-logo"
+            width={180}
+            height={50.99}
```

Comment:
@ntorbinskiy 
![image](https://user-images.githubusercontent.com/46556479/193442624-68a99209-7ea5-4e88-b9a8-0cbb62cb30a7.png)

How do you even get those decimal points?
Can you remove them all?

Reply:
I did incorrect calculations, sorry. In the next commit I'll fix these mistakes